Update enable-betas.mdx (#11252)
PostHog/posthog.com: contents/docs/getting-started/enable-betas.mdx

### How to join an opt-in beta

> To join an opt-in beta you just need to head to the [feature previews menu in-app](http://us.posthog.com/home#panel=feature-previews). It's worth noting that feature previews are on a per-user basis, and will not apply to your entire organization.

Reviewer: ```suggestion
To join an opt-in beta you just need to head to the [feature previews menu in-app](http://us.posthog.com/home#panel=feature-previews). Feature previews are on a *per-user* basis and do not apply to your entire project or organization.
```
